Styles/main.css

```diff
@@ -57,18 +57,21 @@ footer {
     margin-bottom: 2em;
 }
 header {
-    float: right;
-    border: 4px solid rgba(0,0,0,0.1);
+    border-top: 4px solid rgba(0,0,0,0.1);
     margin-left: -2em;
     margin-right: -2em;
 }
+header nav {
+    overflow: auto;
+    margin-left: 2em;
+    margin-right: 2em;
+    margin-top: -4px;
+    border-bottom: 1px dotted rgba(0,0,0,0.3);
+}
 header nav ul{
     float: right;
     padding: 0;
     margin: 0;
-    margin-left: 2em;
-    margin-right: 2em;
-    border-bottom: 1px dotted rgba(0,0,0,0.3);

```

Comment:
These declarations were to apply to `header nav` where I have moved them to above